test/pom.xml

```diff
@@ -47,6 +47,7 @@
     <dependency>
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
+      <version>2.20.0</version>
```

Comment:
@spulci if it didn't had the version before it might be because it is being handled with dependency management.. so we might not need to add the version here..

Reply:
Removed version tag locally; tests with mockito runs fine. I will push it shortly